internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -632,6 +632,15 @@ func StartMachined(_ runtime.Sequence, _ any) (runtime.TaskExecutionFunc, string
 	}, "startMachined"
 }
 
+// StartRuncMemfdBind represents the task to start runc-memfd-bind.
+func StartRuncMemfdBind(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
```

Comment:
I almost feel like it would be easier to run this is a simple controller without any inputs/outputs.

Containerd doesn't depend on this (it needs for the first invocation of runc only, and even if not available, runc would still work).

If we want to make a dependency, we could have used an output resource.

I prefer controllers over anything else, as it's the way forward from the code point of view.